.ci/update_translations_template.md

```diff
@@ -0,0 +1,11 @@
+After merging, all changes to the source language are available to translators at [Transifex](https://www.transifex.com/projects/p/cockatrice/) shortly.
+
+<br>
+
+Updated source strings for translations based on {{ .commit }}:
+- {{ .cockatrice_output }} (Cockatrice)
+- {{ .oracle_output }} (Oracle)
+
+<br>
+
+*This PR is automatically generated and updated by the workflow at `.github/workflows/translations.yml`.*
```

Comment:
is there a specific reason for removing the location of the template here? it'd be useful to know where the template is stored right away.

Reply:
If it would be a an important file that needs frequent updates, I would have kept it that prominently.
As is, it won't see many changes and I think it's no useful information in most cases. The mentioned config file links to it in context as well, so it can still be find relatively well.